dialogs/simplelink.js

```diff
@@ -1,7 +1,13 @@
 ﻿(function (CKEDITOR) {
     'use strict';
 
-    var containsScheme = /:\/\//;
+    function containsScheme (href) {
+        return /^[a-zA-Z]+:/.test(href);
+    }
+
+    function isEmail (href) {
+        return href.indexOf('@') !== -1;
```

Comment:
This isn't a good enough check. An URL can contain a @ character, so I think we should use a simple email regex to make it safer and more correct.